files/zbr-config-gen.sh

```diff
@@ -16,7 +16,8 @@ cat << EndOfMessage
 	  "udid": "${DEVICE_UDID}",
 	  "adb_port": ${ADB_PORT},
 	  "proxy_port": ${PROXY_PORT},
-          "automationName": "${AUTOMATION_NAME}"
+          "automationName": "${AUTOMATION_NAME}",
+          "server_proxy_port": ${SERVER_PROXY_PORT}
```

Comment:
this env var is missed... I'd recommend to declare in Dockerfile near the PROXY_PORT with default value if any

Reply:
done